src/test_utils/kbn_server.js

```diff
@@ -66,6 +67,51 @@ export function createServer(settings = {}) {
   return new KbnServer(defaultsDeep({}, settings, DEFAULTS_SETTINGS));
 }
 
+/**
+ * Creates an instance of KbnServer, including all of the core plugins,
+ * with default configuration tailored for unit tests, and starts es.
+ *
+ * @param  {Object} [settings={}]
+ * @return {KbnServer}
```

Comment:
Let's document what types the setting fields are here

Reply:
Heh. I don't even know what these are. I just pulled this from the `createServerWithCorePlugins` function. But I agree it should be documented. I'll see if I can track down the shape of this thing.